content-src/components/DiscoveryStreamComponents/Hero/Hero.jsx

```diff
@@ -53,23 +55,17 @@ export class Hero extends React.PureComponent {
         source={rec.domain}
         pocket_id={rec.pocket_id}
         bookmarkGuid={rec.bookmarkGuid} />
-      ) : (
-        <PlaceholderDSCard key={`dscard-${index}`} />
       ));
     }
 
-    let list = (
-      <List
-        recStartingPoint={1}
-        data={this.props.data}
-        hasImages={true}
-        hasBorders={this.props.border === `border`}
-        items={this.props.items - 1}
-        type={`Hero`} />
-    );
+    let heroCard = null;
 
-    return (
-      <div className={`ds-hero ds-hero-${this.props.border}`}>
+    if (!heroRec || heroRec.placeholder) {
+      heroCard = (
+        <div className="ds-card placeholder" />
```

Comment:
I believe Hero placeholder should be given some default height, will be good to get wolasi take on it. Here are initial design specs:

https://bugzilla.mozilla.org/show_bug.cgi?id=1529983#c2